addon/lib/metrics.js

```diff
@@ -66,7 +67,7 @@ function makeTimestamp(time) {
 const Metrics = module.exports = {
 
   init: function() {
-    store.browserLoadedTimestamp = Date.now();
+    store.browserLoadedTimestamp = Services.startup.getStartupInfo().process;
```

Comment:
For linting reasons, might want to throw a `// noqa` on this line, since `Services` won't be required.